packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -536,13 +500,13 @@ - (NSDictionary *)encodingMap
 }
 
 UM_EXPORT_METHOD_AS(downloadAsync,
-                    downloadAsyncWithUrl:(NSString *)uriString
+                    downloadAsyncWithUrl:(NSString *)urlString
                     withLocalURI:(NSString *)localUriString
                     withOptions:(NSDictionary *)options
                     resolver:(UMPromiseResolveBlock)resolve
                     rejecter:(UMPromiseRejectBlock)reject)
```

Comment:
Maybe let's gradually make methods names less verbose? 🤔 
Like removing all this `with`, `with`, `with` and have only one of these? 😊